docs/changes.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.4.1
```

Comment:
Maybe bumping to 3.5.0 would make sense as we are adding a feature?
Also make sure to bump the version in `setup.py` too.

Reply:
Makes sense, pushed updated branch.